lib/Map/LeafletVisualizer.ts

```diff
@@ -1226,10 +1226,16 @@ function positionToLatLng(
 }
 
 function hierarchyToLatLngs(hierarchy: PolygonHierarchy) {
-  // This function currently does not handle polygons with holes.
-
+  let holes: any = [];
   const positions = Array.isArray(hierarchy) ? hierarchy : hierarchy.positions;
-  return convertEntityPositionsToLatLons(positions);
+  if (hierarchy.holes.length > 0) {
+    hierarchy.holes.forEach(hole => {
+      holes.push(convertEntityPositionsToLatLons(hole.positions));
+    });
+    return [convertEntityPositionsToLatLons(positions), holes];
```

Comment:
Should this be `return [convertEntityPositionsToLatLons(positions), ...holes];` instead?
That seems to satisfy the type and also matches the [documentation](https://leafletjs.com/reference-1.7.1.html#polygon) (2nd example)

Reply:
Good pickup @na9da - that also fixed my typing problem!!